### DISCUSSION TITLE
Kirin 710

### DISCUSSION THREAD

**Discussion by Wiilly-i-am** [kitsuned/PotatoNV#161] [Ideas]
Hello, Would You be able to add support for Kirin710 CPU's? Including SNE-LX1? if not then why couldn't you add it?

Thanks - willy

**Comment by erikk333** [comment]
https://github.com/map220v/kirin710_bootrom_exploit seems to have a successful implementation for the 710 but idk

**Comment by Wiilly-i-am** [comment]
doesent work for sne lx1 sadly
